### DISCUSSION TITLE
Error while traversing - too many paths watched

### DISCUSSION THREAD

**Discussion by asmod3us** [MobiusSync/MobiusSync#33] [Q&A] [answered]
Hello,

while attempting to sync a folder, I encountered the following error and the folder:

```
Filesystem Watcher Errors:
...

Default Folder	error while traversing /private/var/mobile/Containers/Data/Application/3576FA9B-D610-48C9-89DF-77FFFDB6AFB2/Documents/Sync/Camera Uploads copy: too many paths watched fd=4001
```

Is there anything I can do on the client end to resolve it? Sounds like some ulimit or similar being put into effect?

Thank you!

**Comment by SimonPickup** [comment, marked as answer]
Hi. You should disable "Scanning / Watch for Changes" on this folder as per https://github.com/MobiusSync/MobiusSync/issues/1#issuecomment-739695560
